README.md

```diff
@@ -74,6 +74,68 @@ If you use another CPU and would like to make sure we test it or discuss adding
 an assembly optimized algorithm implementation, please open an issue to discuss
 adding it to our CI.
 
+## Platform Support
+
+AWS-LC supports a number of commonly used CPU architectures. The following is a descriptive
+overview of the platforms available. Tiers are used to indicate how we prioritize our efforts
+for each platform. Regardless of a platform's tier, our goal is to properly support all of the
+available platforms we offer. We gladly accept contributions that improve compatibility or
+performance on these targets.
+
+### Tier 1
+
+Tier 1 platforms are those that are actively tested in our CI pipeline.
+For every pull request, the proposed change is validated to confirm that it successfully builds,
+and all tests pass for each Tier 1 platform. A more complete description
+of our test setup can be found in the
+[CI README](https://github.com/aws/aws-lc/blob/main/tests/ci/README.md#test-locations).
+
+A subset of platforms support building a FIPS variant of the library.
+We do not make any representation or statements regarding such build variants in relation to the
+FIPS 140 standards.
+See [this document](./crypto/fipsmodule/FIPS.md) for more information.
+
+| OS      | CPU     | FIPS<br/>Static | FIPS<br/>Shared |
+|---------|---------|-----------------|-----------------|
+| Linux   | x86     |                 | X               |
+| Linux   | x86-64  | X               | X               |
+| Linux   | aarch64 | X               | X               |
+| Windows | x86-64  |                 | X               |
+| macOS   | x86-64  |                 | X               |
+| macOS   | aarch64 |                 | X               |
+| Android | aarch64 |                 |                 |
+
+### Tier 2
+
+Tier 2 platforms are those that are built and tested on a regular basis, but
+possibly not as part of the AWS-LC CI. These targets are built
+and/or tested in the
+[CI for aws-lc-rs](https://github.com/aws/aws-lc-rs/blob/main/.github/workflows/ci.yml).
+If the build or tests for one of these targets is found to
+be broken, we quickly prioritize the fix.
+
+| OS      | CPU     | Notes |
+|---------|---------|-------|
+| Linux   | arm32   | [1]   |
+| Linux   | ppc64le | [1]   |
+| Linux   | riscv64 | [1]   |
+| iOS     | aarch64 | [2]   |
+| Android | arm32   | [1]   |
+
+(1) Tested in aws-lc-rs CI.<br>
+(2) Built in aws-lc-rs CI.
+
+### Tier 3
+
+Tier 3 platforms are those that our team or our community have put in effort to support.
+The build and/or tests for these platforms will possibly fail. Contributions
+that improve our support for these platforms are appreciated.
```

Comment:
what's the testing standard for PRs addressing Tier 3 support? does the team need to reproduce before merging?

should we provide guidance about how frequently these platforms are built/tested?

Reply:
* In general an issue and its proposed solution (i.e., the PR) should be reproducible and well understood before we merge. But I can imagine there being situations where we're unable to reproduce the problem, but its proposed solution is clearly an improvement to the code. In such cases, the team should evaluate and form a consensus about whether to merge.

* To clarify, I added the sentence "Testing of these platforms might only occur occasionally or when problems are reported."